src/Honeybee.UI/Dialog/Dialog_ModifierSet.cs

```diff
@@ -19,7 +19,11 @@ public IEnumerable<HB.ModifierBase> Modifiers
         {
             get
             {
-                _modifiers = _modifiers ?? this.ModelRadianceProperties.Modifiers.OfType<HB.ModifierBase>().ToList();
+                //_modifiers = _modifiers ?? this.ModelRadianceProperties.Modifiers.OfType<HB.ModifierBase>().ToList();
+                _modifiers = this.ModelRadianceProperties.Modifiers
+                    .OfType<HB.ModifierBase>()
+                    .ToList();
+
```

Comment:
Because if you do this
 `_modifiers = _modifiers != null ? _modifiers : this.ModelRadianceProperties.Modifiers.OfType<HB.ModifierBase>().ToList();`
You never refresh the list. Please see my video in the spreadsheet.

Reply:
It is not because of this null check, I add a commit here to fix the issue you found:
https://github.com/ladybug-tools/honeybee-ui-dotnet/pull/144/commits/cfa75131dd92653ceb528864cdda9e1629e8e927